### PR TITLE
Linted

### PR DESCRIPTION
**Pull request by evaherrada**

### CODE REVIEW

File: examples/featherwing_tft24_simpletest.py [hunk 1]
```diff
@@ -13,11 +13,15 @@
 tft_featherwing = tft_featherwing_24.TFTFeatherWing24()
 
 try:
-    f = open("/sd/tft_featherwing.txt", "w")
+    f = open(  # pylint: disable=unspecified-encoding,consider-using-with
```
Comment (tannewt) [inline examples/featherwing_tft24_simpletest.py]: Why not use `with` as the lint suggests? It'd be:
```
with open("/sd/tft_featherwing.txt", "w") as f:
    f.write("Blinka\nBlackberry G10 Keyboard")
```

**Review by tannewt — CHANGES_REQUESTED** [review]

File: examples/featherwing_tft24_simpletest.py (hunk 1 above)
Comment (evaherrada) [reply examples/featherwing_tft24_simpletest.py]: @tannewt Ah yeah you're right. For some reason I thought it was opening 2 files at the same time.

**Review by tannewt — CHANGES_REQUESTED** [review]
Other files too please.

**Comment by kattni** [comment]
This is no longer needed and will be resolved by #74 - closing.

**Pull request closed without merging**